### PR TITLE
Create kitchen class

### PR DESCRIPTION
**Pull request by 31Nathan**
The kitchen class hosts a thread pool the size of the cook's number

### CODE REVIEW

**Review by Cavonstavant — APPROVED** [review]
LGTM!

File: src/Core/Kitchen/Kitchen.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/*
+** EPITECH PROJECT, 2022
+** plazza
+** File description:
+** Kitchen
+*/
+
```
Comment (osvegn) [inline src/Core/Kitchen/Kitchen.cpp:7]: Need \file

**Review by osvegn — APPROVED** [review]
Need a comment in a file

**Review by osvegn — APPROVED** [review]
LGTM

**Pull request merged**